packages/microsoft-windows-terminal.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,11 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Microsoft Windows Terminal'
+$shortcutDir = ${Env:RAW_TOOLS_DIR}
+$shortcut = Join-Path $shortcutDir "$toolName.lnk"
+
+$AppxPackageName = "Microsoft.WindowsTerminal"
+Remove-AppxPackage -AllUsers -Package (Get-AppxPackage -Name $AppxPackageName -AllUsers -PackageTypeFilter Bundle)
```

Comment:
Does this leave the `microsoft-windows-terminal.vm` package installed? I think that could cause problems.

Related: `chocolatey uninstall --removedependencies`

Reply:
Hmmm, I will double check on that.

Would using `chocolatey uninstall --removedependencies` be an acceptable method I can use here?

Reply:
I think other metapackages just keep the dependencies installed and the user would run `chocolatey uninstall "microsoft-windows-terminal.vm" --removedependencies` if they want to uninstall them. Uninstalling is best effort. So we don't have very complicated uninstallers.